### ISSUE TITLE
can't remove a CT

### ISSUE THREAD

**Issue by fab33** [LINBIT/linstor-proxmox#4]
When I want to remove a CT I have this error : 
```
usage: linstor [-h] [--version] [--no-color] [--no-utf8] [--warn-as-error]
[--controllers CONTROLLERS] [-m] [-t TIMEOUT]
[--disable-config]
{controller, encryption, error-reports, help, interactive,
list-commands, node, resource, resource-definition, snapshot,
storage-pool, storage-pool-definition, volume-definition} ...
TASK ERROR: error with cfs lock 'storage-drbdstorage': Could not remove vm-106-disk-1: linstor: error: unrecognized arguments: -q
```

**Comment by acidrop** [comment]
Should be fixed by the last commit, today.

**Comment by rck** [comment]
Duplicate of #3 
This should be fixed with `2.9.1`, and packages should already be in the repositories.
